### DISCUSSION TITLE
Visual Studio Setting Sync always on when running new Codespace

### DISCUSSION THREAD

**Discussion by smajchrz** [community/community#18030] [Codespaces] [answered]
My settings like extensions and fonts settings are always synced with Codespace, whether I run it from a branch with or without .devcontainer file. I am syncing my local VS setting using my GitHub account. 

Is there any way to turn such sync off during Codespace boot?

**Comment by spaltrowitz** [comment, marked as answer]
Hi @smajchrz, thanks for sharing this feedback. A feature related to this feedback has now been shipped! 🚢 https://github.blog/changelog/2023-05-04-codespaces-settings-sync-updates/
